src/models/visualNovel.ts

```diff
@@ -0,0 +1,285 @@
+import { SpoilerLevel } from '../utils/types/spoilerLevel';
+import { VisualNovelLinks } from '../utils/types/visualNovelLinks';
+import { ImageFlagging } from './imageFlagging';
+
+/**
+ * Represents anime related to the VN.
+ */
+export interface VisualNovelRelatedAnime {
+
+  /**
+   * AniDB ID.
+   */
+  id: number;
+
+  /**
+   * AnimeNewsNetwork ID.
+   */
+  annId: number | null;
+
+  /**
+   * AnimeNfo ID.
+   */
+  nfoId: string | null;
+
+  /**
+   * Title in romaji.
+   */
+  titleRomaji: string | null;
+
+  /**
+   * Title in kanji.
+   */
+  titleKanji: string | null;
+
+  /**
+   * Date in which the anime was aired.
+   */
+  year: Date | null;
```

Comment:
Maybe rewrite it to `date when the anime was first aired`? And rename property accordingly.